db/migrate/20150128155644_allow_null_for_adviser_covers_whole_of_uk.rb

```diff
@@ -0,0 +1,5 @@
+class AllowNullForAdviserCoversWholeOfUk < ActiveRecord::Migration
+  def change
+    change_column :advisers, :covers_whole_of_uk, :boolean, null: true, default: nil
```

Comment:
It's probably not good to use db defaults for an application requirement.  Probably wants to be done in an ``after_initialize`` callback.